server/src/domain/auth/auth.dto.ts

```diff
@@ -1,19 +1,14 @@
-import { UserEntity, UserTokenEntity } from '@app/infra/entities';
+import { APIKeyEntity, SharedLinkEntity, UserEntity, UserTokenEntity } from '@app/infra/entities';
 import { ApiProperty } from '@nestjs/swagger';
 import { Transform } from 'class-transformer';
 import { IsEmail, IsNotEmpty, IsString, MinLength } from 'class-validator';
 
-export class AuthUserDto {
-  id!: string;
-  email!: string;
-  isAdmin!: boolean;
-  isPublicUser?: boolean;
-  sharedLinkId?: string;
-  isAllowUpload?: boolean;
-  isAllowDownload?: boolean;
-  isShowMetadata?: boolean;
-  accessTokenId?: string;
-  externalPath?: string | null;
+export class AuthDto {
+  user!: UserEntity;
+
+  apiKey?: APIKeyEntity;
+  sharedLink?: SharedLinkEntity;
+  userToken?: UserTokenEntity;
 }
```

Comment:
This is the bulk of the change, the rest of it is just renaming/updating accessors.